lib/ansible/modules/windows/win_disk_facts.ps1

```diff
@@ -0,0 +1,132 @@
+#!powershell
+
+# Copyright: (c) 2017, Marc Tschapek <marc.tschapek@itelligence.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#Requires -Module Ansible.ModuleUtils.Legacy
+$ErrorActionPreference = "Stop"
+Set-StrictMode -Version 2.0
+
+# Create a new result object
+$result = @{
+    changed = $false
+    ansible_facts = @{
+        ansible_disk = @{
+        } 
+    }
+}
+
+# Search disks
+try {
+    $disk = Get-Disk
+} catch {
+    Fail-Json -obj $result -message "Failed to search the disks on the target: $($_.Exception.Message)"
+}
+if ($disk) {
+    [string]$diskcount = $disk | Measure-Object | Select-Object  -ExpandProperty Count
```

Comment:
Is there a reason why we cast this as a string, we should be returning it as an integer on the return value.

Reply:
Changed